### PR TITLE
fixes #45 - Fixes parts of SVG icon problem

### LINKED ISSUE #45: Correct position of arrow icon, transformation on hover & hide icon in mobile view
screen shot 2017-11-09 at 7 53 03 pm

The icon in the "report bug" button needs some love.

- [ ] the icon size is too small and needs to be fixed, so it aligns with the letters
screen shot 2017-11-09 at 7 58 23 pm

- [ ] the icon should transform +90° on hover
screen shot 2017-11-09 at 7 58 26 pm

- [ ] the width of the button should be wider, so it stays in one line on mobile

CURRENT:
screen shot 2017-11-09 at 7 59 36 pm

SHOULD:
screen shot 2017-11-09 at 7 59 46 pm

### PR DESCRIPTION
**Pull request by madeleine-neumann**
Some parts of the issue can't be fixed until the SVG Icon Problem is fixed.
@zoepage R and maybe new Issue with SVG Icon  size/positioning problem?

### CODE REVIEW

**Comment by zoepage** [comment]
related to #20.

File: docs/css/button.css [hunk 1]
```diff
@@ -35,13 +35,30 @@
   fill: var(--color-first);
 }
 
+.button-hero .icon-middle{
+  -webkit-transform: rotate(-90deg);
+  -moz-transform: rotate(-90deg);
+  -o-transform: rotate(-90deg);
+  -ms-transform: rotate(-90deg);
+  transform: rotate(-90deg);
```
Comment (kotzendekrabbe) [inline docs/css/button.css]: we don't need prefixes for properties.

File: docs/css/button.css [hunk 2]
```diff
@@ -35,13 +35,30 @@
...
+}
+
+.button-hero:hover .icon-middle{
```
Comment (kotzendekrabbe) [inline docs/css/button.css]: the icon need a vertical-align middle + the text right after it also

**Comment by madeleine-neumann** [comment]
@kotzendekrabbe Review again?

**Review by kotzendekrabbe — DISMISSED** [review]

**Comment by kotzendekrabbe** [comment]
@madeleine-neumann can you please resolve the conflicts :)

**Pull request merged**